src/components/addSourceWizard/hardcodedComponents/oci/costManagement.js

```diff
@@ -0,0 +1,232 @@
+        {intl.formatMessage({
+          id: 'cost.oci.compartmentIdStepTitle',
+          defaultMessage: 'Global compartment-id',
+        })}
+      </Title>
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'cost.oci.compartmentId.description',
+          defaultMessage:
+            'To collect and store the information needed to manage your costs, you need to first find your Global compartment-id.',
+        })}
+      </Text>
+      <TextList className="pf-u-ml-0 pf-u-pl-md" component={TextListVariants.ol}>
+        <TextListItem className="pf-u-mb-lg">
+          {intl.formatMessage({
+            id: 'cost.oci.compartmentId.subtitle1',
+            defaultMessage: 'In the Oracle Cloud shell, copy and paste this command into the terminal to list your compartments',
+          })}
+          <ClipboardCopy className="pf-u-mt-sm">oci iam compartment list</ClipboardCopy>
+        </TextListItem>
+        <TextListItem>
+          {intl.formatMessage({
+            id: 'cost.oci.compartmentId.subtitle2',
+            defaultMessage: 'Enter the name of the your global compartment-id (tenant-id)',
+          })}
+          {fields.map((field) => (
+            <div className="pf-u-mt-sm" key={field.name}>
+              {renderForm([field])}
+            </div>
+          ))}
+        </TextListItem>
+      </TextList>
+    </TextContent>
+  );
+};
+
+CompartmentId.propTypes = {
+  fields: PropTypes.array.isRequired,
+};
+
+export const PolicyCompartment = ({ fields }) => {
+  const intl = useIntl();
+
+  const { renderForm, getState } = useFormApi();
+  const values = getState().values;
+
+  return (
+    <TextContent>
+      <Title headingLevel="h1">
+        {intl.formatMessage({
+          id: 'cost.oci.policyCompartmentStepTitle',
+          defaultMessage: 'Create new policy and compartment',
+        })}
+      </Title>
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'cost.oci.policyCompartment.description',
+          defaultMessage:
+            'In the Oracle Cloud shell, copy and paste these commands to create your cost and usage reports policy, and new cost compartment.',
+        })}
+      </Text>
+      <TextList className="pf-u-ml-0 pf-u-pl-md" component={TextListVariants.ol}>
+        <TextListItem className="pf-u-mb-lg">
+          {intl.formatMessage({
+            id: 'cost.oci.policyCompartment.subtitle1',
+            defaultMessage: 'Create cost and usage reports policy using the following command',
+          })}
+          <ClipboardCopy className="pf-u-mt-sm" variant={ClipboardCopyVariant.expansion}>
+            {`oci iam policy create --compartment-id ${values?.application?.extra?.compartment_id} --description "test" --name "test" --statements '["define tenancy usage-report as ocid1.tenancy.oc1..aaaaaaaaned4fkpkisbwjlr56u7cj63lf3wffbilvqknstgtvzub7vhqkggq","endorse group Administrators to read objects in tenancy usage-report"]'`}
```

Comment:
Also, this `aaaaaaaaned4fkpkisbwjlr56u7cj63lf3wffbilvqknstgtvzub7vhqkggq` looks a bit weird, I know it is in the mockups but we should maybe try to find out how it should be cleaned a bit and more clear for the user